### PR TITLE
Parkourtest

### LINKED ISSUE #207: fix for Organisms = list
https://github.com/maxplanck-ie/dissectBCL/blob/c06b512a56b781d2c1ec7bcd0d6f3d6a00f3b406/src/dissectBCL/misc.py#L513C26-L513C49

https://github.com/maxplanck-ie/dissectBCL/blob/c06b512a56b781d2c1ec7bcd0d6f3d6a00f3b406/src/dissectBCL/fakeNews.py#L446

### PR DESCRIPTION
**Pull request by thomasmanke**
this fixes #207
This was tested with **parkour-test** which already returns a list for organism
it could become production together with the new BRB once **parkour** has updated .

### CODE REVIEW

**Comment by thomasmanke** [comment]
unsure if to use main or prod for PR

File: src/dissectBCL/misc.py [hunk 1]
```diff
@@ -510,7 +510,7 @@ def multiQC_yaml(flowcell, project, laneFolder):
     ))
     # Organisms
     orgs = ', '.join(list(
-        ssdf["Organism"].fillna('None').unique()
+        ssdf["Organism"].str[0].fillna('None').unique()
```
Comment (adRn-s) [inline src/dissectBCL/misc.py:513]: I'm a bit puzzled by this, is it to get the "H." in the full string, e.g. "H. sapiens (GRCh37 / hg19)"? Can you share how 1 of the rows in `ssdf` look? Only if you have it around, on the tmux session... Else I can try thinking harder.. just to be sure we are on the same page, here (folowing the example) we want the string "hg19" (on parkour DB a.k.a. "label"). right?

Comment (thomasmanke) [reply src/dissectBCL/misc.py:513]: No, but unique() would not work with a list, i.e the result of ssdf["Organism"].fillna('None')

Comment (thomasmanke) [reply src/dissectBCL/misc.py:513]: ["human", None, None]

**Pull request merged**